src/less-style/lib/variables.less

```diff
@@ -11,3 +13,38 @@
 @grayDarker:            #2C2C2C;
 @blueLight:             #2787d0;
 
+// Main Colors [NEW]
```

Comment:
they're in the bootstrap 3 styles...feel free to add them to less/includes/variables.less if all that is automated for now